src/App.jsx

```diff
@@ -14,15 +14,26 @@ import BillingAddress from "./components/BillingAddress/BillingAddress";
 import LandingMain from "./containers/LandingMain/LandingMain";
 import ContactListPage from "./components/ContactListPage/ContactListPage";
 import LiveRates from "./components/LiveRates/LiveRates";
+import CurrencyConverterContainer from "./containers/CurrencyConverterContainer/CurrencyConverterContainer";
+import currency from './data/currency';
+
 
 const App = () => {
   const [searchTerm, setSearchTerm] = useState("");
-
+  const [baseCurrency, setBaseCurrency] = useState(currency[0]);
```

Comment:
Hardcoded values will need to be refactored afterwards to allow use to choose another currency

Reply:
-Consider a drop down on the to currency.
-The base currency is determine by the Live Rate with 1
- Functionality for the converter needs to be revised. For example, If bank account (i.e., my base currency ) is in USD, and I need to send EUR 75.14, I need to be able to switch the currencies, but beep my base currency intact (i.e., FROM currency)